### ISSUE TITLE
Better handle CSV import via Places Import

### ISSUE THREAD

**Issue by justin-bits** [ngageoint/opensphere#740]
CSV imports via Layers>Places are currently handled non-intuitively.  KML is the expected format and is handled correctly.  GeoJSON and SHP are not expected, but imported easily as features.  CSV import throws a warning indicating that the file type could not be detected.  Only after selecting CSV from the drop down can the file be imported as features.

- Adjust file format detection to work with CSV (as it already does with GeoJSON and SHP)
- [optional] Notify the user that the file contents will be imported as features, where the option then exists to save those features as places, OR ...
- [optional] Give the user the ability to drop the contents in directly as places

**Comment by bradh** [comment]
This doesn't appear to be CSV specific. I see the `AnyTypeImportUI` being used for GeoJSON (e.g. a local copy of https://docs.mapbox.com/help/data/stations.geojson) and SHP (e.g. an exported file from OpenSphere).

It looks like `os.ui.im.ImportManager` only knows about UIs for `application/vnd.google-earth.kml+xml` or `application/vnd.google-earth.kmz` (which go to `plugin.places.KMLPlacesImportUI`) and `text/xml; subtype=filter` (which goes to `os.filter.im.OSFilterImportUI`). Not sure that last one is going to work in Places Import, but that is probably a separate issue.
